lib/cloudfront-signer.rb

```diff
@@ -266,6 +263,15 @@ def self.epoch_time(timelike)
         end
       end
 
+      def self.build_resources(resource, conditions)
```

Comment:
Don't you think `build_statements` would be a more appropriate name? It seem you are building an array of statements, all with Resource and Condition.